### ISSUE TITLE
Broker API

### ISSUE THREAD

**Issue by akelley** [dwmkerr/terraform-aws-openshift#26]
Does the broker api not get created/started during the initialization process on the master?  I get a 403 when trying to call the endpoint `/broker/rest/api`.  Is there a command to start up the broker or can the script be modified to include that service?

**Comment by dwmkerr** [comment]
Hi there @kelleyaj do you mean the new tech preview service broker?

https://blog.openshift.com/whats-new-openshift-3-6-service-catalog-brokers-tech-preview/

If so, it's not set up by default, but I could potentially add it as a recipe you can mix into the project!

**Comment by kelleyaj** [comment]
Yes sorry, I now see that it is in the next release 3.6 tech preview.  I have a external Jenkins instance where I am trying to utilize open shift for application deployment, it uses a plugin that needs a broker url to call.  See here - https://blog.openshift.com/deploy-openshift-external-jenkins-instance/ 
That would be great if it can be added!  Sorry I would do a pull myself but I am still coming up to speed on terraform scripting and open shift, using this great project here to learn.  Thanks!

**Comment by dwmkerr** [comment]
Looks really interesting! I'll take a look at the broker set up and see if it can be included somehow as an optional component :)
